pkg/util/helm_helpers.go

```diff
@@ -299,6 +259,30 @@ func GetWithHelm3(releaseName, namespace, kubeConfig string) (*release.Release,
 	return nil, fmt.Errorf("unable to find release '%s' in namespace '%s'", releaseName, namespace)
 }
 
+// GetBlackDuckChartAndReleaseWithHelm3 checks whether an file exist in the chart. If exist, it will convert the file to map and return the map
+func GetBlackDuckChartAndReleaseWithHelm3(namespace, kubeConfig, chartURL, fileName string) (map[string]interface{}, error) {
```

Comment:
Can we rename this function to something more clear - like GetValuesFromFileInHelmChart() ?